### ISSUE TITLE
Timeboxed investigation of using the Content API of stories listing/lists

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#11146]
`/stories`, `/homepage`, there might be more?

We want to try using the Content API to display lists of stories, and we need to see if it's a quick job or if changes are needed on the API side.

This will unblock #11127's webcomics section.

**Comment by LaurenFBaily** [comment]
Agree time for timebox - half day or day?

**Comment by davidpmccormick** [comment]
## Homepage
- ✔️ The homepage lists the last four stories of any format, so this should be a straightforward call to the `/articles` endpoint.

## Stories
- ✔️ The top section of the /stories page filters out comics and displays the last five – this should also be straightforward
- ✖️ The next section down on the homepage is a curated list managed through the ['stories landing page' type in Prismic](https://wellcomecollection.prismic.io/builder/pages/Yx8r4xEAADZUti36?fallback=ayUzRHN0b3JpZXMtbGFuZGluZyUyNnclM0RzdG9yaWVzJTI2YiUzRHdvcmtpbmclMjZjJTNEcHVibGlzaGVkJTI2bCUzRGVuLWdi&s=published&section=Featured+stories%2Fseries), and I don't think this could be swapped out for content from the Content API
- ✖️ The 'comics' section is three cards for comic serials, rather than individual comics, not including any comics from the current serial – I don't believe we can query for serials from the Content API currently
- ✖️ The 'books' section is also managed through the 'stories landing page' in Prismic so not possible to switch out for the Content API I don't think (and even if it weren't I don't think the API currently handles books)
- ✔️ The 'you may have missed' section is the last 6–11 (non-comic) articles so should be possible as part of the same query as for the top section outlined above

**Comment by rcantin-w** [comment]
>  The 'comics' section is three cards for comic serials, rather than individual comics, not including any comics from the current serial – I don't believe we can query for serials from the Content API currently

You're right, we don't allow that as `series` are not in Elasticsearch, so it would probably be about adding that in. 
I wonder if we could consider it a non-blocker for https://github.com/wellcomecollection/wellcomecollection.org/issues/11127. The Body Squabble series would show there for a while, but maybe it's not a bad thing?

>  The 'books' section is also managed through the 'stories landing page' in Prismic so not possible to switch out for the Content API I don't think (and even if it weren't I don't think the API currently handles books)

That's true we don't pull `books` in, so I think out of scope

Agree with all your other points! So it'd be about replacing:
- homepage stories list
- the top section of the /stories page
- the 'you may have missed' section of the /stories page

And we think the switch could be done quite quickly/without issues then?

**Comment by pollecuttn** [comment]
To be discussed at planning 2 October

**Comment by rcantin-w** [comment]
Added warnings in https://github.com/wellcomecollection/wellcomecollection.org/issues/11127 and details in https://github.com/wellcomecollection/wellcomecollection.org/issues/11226

I think we could close this @davidpmccormick @LaurenFBaily @pollecuttn ?
